lib/rspec/snapshot/utils.rb

```diff
@@ -0,0 +1,75 @@
+require 'fileutils'
+require 'oj'
```

Comment:
[oj](https://github.com/ohler55/oj) is the best ruby json serializer / deserializer I've found.

Reply:
I like and use oj, but is it possible be better to make him optional?